helix-view/src/input.rs

```diff
@@ -6,10 +6,13 @@ use std::fmt;
 
 pub use crate::keyboard::{KeyCode, KeyModifiers};
 
-#[derive(Debug, PartialOrd, PartialEq, Eq, Clone, Copy, Hash)]
```

Comment:
Because the paste data is on the heap, we have to lose this Copy. That's why there's all this introduction of references through the rest of the PR.